apps/src/applab/designMode.js

```diff
@@ -308,6 +308,23 @@ designMode.updateProperty = function(element, name, value) {
       break;
     case 'readonly':
       element.setAttribute('contenteditable', !value);
+    case 'is-default':
+      element.setAttribute('is-default', value);
```

Comment:
I think we want this to be data-is-default http://stackoverflow.com/a/1735268/2506748